src/getDefaultAlign.js

```diff
@@ -1,6 +1,8 @@
+// @flow
 import type { PositionT } from './flowTypes'
 
-const getDefaultAlign = (position: PositionT) =>
+const getDefaultAlign = (position: PositionT): PositionT =>
+  // $FlowIgnore: the resulting string will be one of the PositionT values, trust me ;)
   position
```

Comment:
`flow` will pass here if you construct the string first and then check that is part of the correct domain with `array.find` on an array that contains all possible values. That might seem cumbersome here but would guarantee type safety plus remove the need for `FlowIgnore`. 

While you're at it: The npm script should probably use `flow check` instead of just `flow`.

Reply:
And fix: https://github.com/signavio/react-stick/issues/9 :P